powerline/bindings/lemonbar/powerline-lemonbar.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
```

Comment:
actually that's right, it never occurred to me because I had the old one laying around for a long time and wrote the multimonitor one from scratch just now